sdk/keyvault/azure-security-keyvault-keys/CMakeLists.txt

```diff
@@ -90,7 +100,7 @@ if (BUILD_PERFORMANCE_TESTS)
 endif()
 
 if(BUILD_SAMPLES)
-  add_subdirectory(sample)
+  add_subdirectory(test/samples)
```

Comment:
Why are we moving `samples` to be under `test/`? I am not sure they belong there. Aren't these customer facing, acting as documentation on how to use the client?

Reply:
following .NET and other languages structure.  
.../samples -> Documentation with code snippets to .../test/samples. There is an .md file with explanation and code about the sample and a link to the sample source code.
.../test/samples -> Compilable and runnable sample referenced by an .md file from ../samples